Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocCommandColumnRendererTest.cs

```diff
@@ -122,7 +123,7 @@ public void RenderCommandIconCell ()
       Column.Icon.Width = new Unit(16, UnitType.Pixel);
       Column.Icon.Height = new Unit(16, UnitType.Pixel);
 
-      IBocColumnRenderer renderer = new BocCommandColumnRenderer(new FakeResourceUrlFactory(), RenderingFeatures.Default, _bocListCssClassDefinition);
+      IBocColumnRenderer renderer = new BocCommandColumnRenderer(new FakeResourceUrlFactory(), RenderingFeatures.Default, _bocListCssClassDefinition, SafeServiceLocator.Current.GetInstance<IFallbackNavigationUrlProvider>());
```

Comment:
Here should be DefaultFallbackNavigationUrlProvider / FakeFallbackNavigationUrlProvider